packages/protocol/thread/pause.ts

```diff
@@ -242,11 +248,20 @@ export class Pause {
       if (!this.frames.has(f.frameId)) {
         this.frames.set(f.frameId, f as WiredFrame);
       }
+      if (!this.rawFrames.has(f.frameId)) {
+        const rawFrame = cloneDeep(f);
```

Comment:
Is this cloning step necessary because of the fronts and mutation?

Reply:
Yes.

Reply:
BUT HOPEFULLY NOT FOR VERY LONG! :)